public/javascripts/dashboard.js

```diff
@@ -0,0 +1,19 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+    const cookBookIcon = document.getElementById('open-book')
+    const logOutIcon = document.getElementById('logout-user-button')
+    const recipeCard = document.querySelectorAll('card')
+    
+
+    cookBookIcon.addEventListener('click', (event) => {
+        window.location.href = "http://localhost:8080/cookbooks"
+    })
```

Comment:
This can be accomplished with an `a` tag, so please by sure to remove this code and use an `a` tag in the template instead.